setup-prompts.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
+
+module.exports = [{
+  type: "input",
+  name: "clientData",
+  message: "Client data directory",
+  default: "C:/Program Files (x86)/World of Warcraft/Data",
+  validate: function(value) {
+    try {
+      if (fs.lstatSync(value).isDirectory()) {

```

Comment:
Hadn't thought of this one, so I didn't add any runtime checks. Not sure how Wowser loads the resources from there I guess if it tried to access non-existing directory it would fail, but then again maybe it couldn't be deleted if its used?